components/core/src/crypto/keys/box_key_pair.rs

```diff
@@ -54,22 +61,107 @@ impl<'a> WrappedSealedBox<'a> {
         str::from_utf8(bytes).map(Cow::Borrowed)
                              .map(WrappedSealedBox)
     }
+
+    // Return the metadata and encrypted text from a secret payload.
+    // This is useful for services consuming an encrypted payload and need to decrypt it without
+    // having keys on disk
+    pub fn secret_metadata<'b>(&'b self) -> Result<BoxSecret<'b>> {
```

Comment:
I think this lifetime can be elided `pub fn secret_metadata(&self) -> Result<BoxSecret>`

Reply:
Fortunately this type will be totally removed in some soon-to-be-pushed commits :partying_face: